iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/storable_function.inl

```diff
@@ -34,11 +34,14 @@ storable_function<S, signature<ReturnType, Args...>>::storable_function(const Fu
 template <typename S, typename ReturnType, typename... Args>
 storable_function<S, signature<ReturnType, Args...>>::storable_function(ReturnType (*function)(Args...)) noexcept
 {
-    m_invoker = invokeFreeFunction;
-    m_callable = reinterpret_cast<void*>(function);
-    m_operations.copyFunction = copyFreeFunction;
-    m_operations.moveFunction = moveFreeFunction;
-    // destroy is not needed for free functions
+    if (function)
```

Comment:
Was needed since it otherwise tries to invoke a nullptr. This way if we construct it from a nullptr it is not callable and hence correct (the variables we do not set directly are initialized with nullptr).